### DISCUSSION TITLE
How can I type hex codes into `insert-char` without leading zeroes while running vertico-mode?

### DISCUSSION THREAD

**Discussion by Bujiraso** [minad/vertico#491] [Q&A] [answered]
Running Emacs with `emacs -Q` allows you to run `M-x insert-char` and type, say, `d2` to get `Ò`. You don't have to type the leading zeroes (that is, `0000d2`).
Once you install vertico (I did so in the same session), you can no longer type such a hex code to get such a character.

![image](https://github.com/minad/vertico/assets/1975771/36c47cf3-254e-4ea6-8d5f-45a5db997b86)

How can I configure vertico to permit me to use such addressing?
A worst-case solution would be to ignore vertico when `insert-char`ing. A non-solution is to type the leading zeroes each time, which does bypass this problem.

**Comment by minad** [comment]
See https://github.com/minad/vertico?tab=readme-ov-file#key-bindings about prompt selection and submitting the input.

**Reply by Bujiraso** [reply, marked as answer]
Thank you, `M-RET` does indeed do what I want!
